assets/css/src/amp-stories.css

```diff
@@ -34,7 +34,7 @@ amp-story-grid-layer[template="fill"] .wp-block-image {
 	font-size: 18px;
 	margin: 0;
 	max-height: calc(100% - 24px);
-	padding: 12px 24px;
+	padding: 7px 24px;
```

Comment:
This reflects the button in FE, or?

Reply:
Yep, this change should remove the difference between the editor and FE. Are you thinking that perhaps it would be better to change it in the editor instead? The 12px seemed too much when testing out with the added amp-fit-text.

Reply:
I was just curious. Seems legit to me :)